igvm_defs/src/lib.rs

```diff
@@ -478,11 +478,11 @@ pub struct TdxPolicy {
 
 /// This region describes VTL2.
 pub const IGVM_VHF_RELOCATABLE_REGION_IS_VTL2: u8 = 0x1;
-/// RIP for the specified VP and VTL should be adjusted by the amount this
-/// region was relocated.
+/// The starting executable address for the specified VP and VTL should be
+/// adjusted by the amount this region was relocated (RIP on x64, PC on arm64).
 pub const IGVM_VHF_RELOCATABLE_REGION_APPLY_RIP: u8 = 0x2;
```

Comment:
lets rename this constant to maybe APPLY_PC or something like apply_instr_reg?

Reply:
Wouldn't that be a breaking change? Is that something we do?

Reply:
Hmm... I guess it would be, but we're not changing the value just the name. I think it's a breaking change to say this is supported in v2 anyways, since it wasn't previously supported on anything except x64. 

We can publish a new crate version for the rename.

Reply:
I added IGVM_VHF_RELOCATABLE_REGION_APPLY_START_ADDRESS and also defined IGVM_VHF_RELOCATABLE_REGION_APPLY_RIP for compat purposes.

Reply:
I'd probably just call it `IGVM_VHF_RELOCATABLE_REGION_APPLY_PC` and have the second definition for RIP for compat. Given that "program counter" is somewhat architecture agnostic. `start_address` doesn't really apply neatly to a register, since these are flags for what registers should be modified.